### DISCUSSION TITLE
Loading component does not show

### DISCUSSION THREAD

**Discussion by nblkmal** [saadeghi/daisyui#1961] [General]
![Screenshot 2023-06-09 at 2 44 03 PM](https://github.com/saadeghi/daisyui/assets/72432370/57d13d68-1e14-44e3-9ea8-9a88d7e72255)

As you can see, im using the exact example from the docs of using the button with loading spinner, however it does not show in the ui, any suggestion?

Here is the docs link for reference : https://daisyui.com/components/loading/

**Comment by MMore** [comment]
Have you updated Daisy to the current version? Loading became available in v3.

**Reply by nblkmal** [reply]
Omg silly me, thank you so much! It is in the change log https://daisyui.com/docs/changelog/#300-2023-06-01
